packages/expo-optimize/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "expo-optimize",
+  "version": "0.0.0",
+  "main": "build",
+  "preferGlobal": true,
+  "keywords": [
+    "expo",
+    "sharp",
+    "optimize",
+    "react-native",
+    "react-native-web"
+  ],
+  "description": "Compress the assets in your project",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/expo/expo-cli.git",
+    "directory": "packages/expo-optimize"
+  },
+  "author": "Expo <support@expo.io>",
+  "license": "MIT",
+  "bin": {
+    "expo-optimize": "./build/index.js"
+  },
+  "files": [
+    "build"
+  ],
+  "scripts": {
+    "prepare": "yarn run clean && yarn run build:prod",
+    "lint": "eslint .",
+    "watch": "yarn run build -w",
+    "build": "ncc build ./src/index.ts -o build/ --external @expo/image-utils",
```

Comment:
Can you explain how `ncc` helps here, if we leave `@expo/image-utils` as an external? Presumably `npx` will still have to download and install `@expo/image-utils` anyway?

Reply:
Because this package doesn't have an accessible API we don't need a types or source map file, `npx` will download the file significantly faster this way. It's using Webpack to compile the TS results down to one file, I was going to build my own implementation but this seemed to have everything I needed.